src/bitcoin-client.js

```diff
@@ -33,6 +41,11 @@ function getAddressFromPublicKey(publicKey) {
   return /** @type string */ (payment.address);
 }
 
+const getFastestFee = () =>
+  USE_BLOCKSTREAM
+    ? bitcoin.fees.getFeeEstimates().then(get("1"))
+    : bitcoin.fees.getFeesRecommended().then(get("fastestFee"));
```

Comment:
I feel like this should be solved inside `esploraJs` and exposing only one function for fees, instead of 2 😅  But all in all, I understand if you don't want to take extra effort to refactor that.

Reply:
The problem with that is the lib would need to know what API/URL is pointing too and this is known by the lib user (`baseUrl` is injected).

We could also move the URLs and this logic inside, then pass a flag or string to use Mempool or Blockstream. But didn't want to add complexity in there at the moment.